PLF-6493 [Profile Page] Buttons Connect/disconnect and Chat in user profile acting strangely

dongpd opened the pull request:
Analysis: css of user-actions class effects all elements inside when hovering
Solution: Use javascript to let hover only to work on buttons

minhdv commented on weemo-extension-webapp/src/main/java/org/exoplatform/portlet/videocall/ResponseFilter.java:
```diff
@@ -26,11 +36,36 @@ public void doFilter(RenderRequest request, RenderResponse response, FilterChain
+      br.close();
+      return ! (stringBuilder.toString().toUpperCase().equals(INVALID_WEEMO_JS_MESSAGE.toUpperCase()));
+    } catch (Exception e) {
+      e.printStackTrace();
```
Please use LOG here instead of print stacktrace

tungdt replied:
Fixed: use LOG instead of print stacktrace

trangvh commented on weemo-extension-webapp/src/main/java/org/exoplatform/portlet/videocall/ResponseFilter.java:
```diff
@@ -26,11 +38,36 @@ public void doFilter(RenderRequest request, RenderResponse response, FilterChain
       jQuery1.setAttribute("type", "text/javascript");
-      jQuery1.setAttribute("src", "https://download.rtccloud.net/js/webappid/"+weemoKey+" ");
+      if(checkWeemoJavaScript()){
+        jQuery1.setAttribute("src", "https://download.rtccloud.net/js/webappid/"+weemoKey+" ");
```
the URL should be declared as a static final variable.

dongpd committed:
PLF-6493 [Profile Page] Buttons Connect/disconnect and Chat in user profile acting strangely

Analysis: css of user-actions class effects all elements inside when hovering
Solution: Use javascript to let hover only to work on buttons
